ush/parsing_namelists_CICE.sh

```diff
@@ -19,12 +19,15 @@ cat > ice_in <<eof
   , use_restart_time = $USE_RESTART_TIME
   , restart_format = 'nc'
   , lcdf64         = .false.
-  , restart_dir    = './restart/'
+  , numin          = 21
```

Comment:
Should we go ahead and update the format here be updated to match that of https://github.com/ufs-community/ufs-s2s-model/blob/s2s_prototype5.0/parm/ice_in_template ? so no more , at the beginning of every line?

Reply:
This has been addressed